Tests/Functional/app/ViewResponseListener/config.yml

```diff
@@ -13,8 +13,6 @@ fos_rest:
         formats:
             xml:  true
             json: true
-        templating_formats:
-            html: true
     format_listener:
         rules:
             - { path: ^/, priorities: [ html, json, xml ], fallback_format: ~, prefer_extension: true }
```

Comment:
can the `html` format be removed from here too?

Reply:
I tried to do that initially but that changed the status code of the response. However, deprecating the force redirects option as suggested by @lsmith77 solve this. We now have to dedicated test cases for this feature.